docs/en/latest/terminology/plugin.md

```diff
@@ -278,4 +278,4 @@ If a configured Plugin is disabled, then its execution will be skipped.
 
 ### Hot reload in stand-alone mode
 
-For hot-reloading in stand-alone mode, see the plugin related section in [stand alone mode](../deployment-modes.md#stand-alone).
+For hot-reloading in stand-alone mode, see the plugin related section in [stand alone mode](https://apisix.apache.org/docs/apisix/deployment-modes/#standalone).
```

Comment:
The URL can be updated to relative path → `... section in [stand alone mode](../deployment-modes/#standalone).`

cc: @navendu-pottekkat

Reply:
ok.